example/README.md

```diff
@@ -0,0 +1,10 @@
+## <a name="example"></a> Example Application
+
+You can quickly run and interact with some code by looking at the files in the `/example` folder. You run the example with the following commands:
+
+```bash
+$ npm ci
+$ npm run example:start
+```
+
+This runs a simple server at `http://localhost:3000/` that serves the static `example/index.html` file. This allows it easily interact with Metamask. You can edit the `example/index.html` file to try different code.
```

Comment:
> http://localhost:3000/

Should be port 30000 here :)

Reply:
Updated!

Reply:
@oed also update the introductory how-to from the main README "interact with the example code in this directory" etc